tests/TestCase.php

```diff
@@ -3,15 +3,31 @@
 namespace Jevets\Kirby\Flash\Tests;
 
 use Mzur\Kirby\DefuseSession\Defuse;
+use Jevets\Kirby\Flash;
 
 class TestCase extends \PHPUnit\Framework\TestCase
 {
     /**
      * Default preparation for each test.
      */
-    public function setUp()
+    public function setUp(): void
     {
         parent::setUp();
         Defuse::defuse();
+        $this->nextPageLoad();
+        // run two full page loads to ensure each test is isolated
+        $this->nextPageLoad();
```

Comment:
Why are two required?

Reply:
This ensures each test is fully isolated.

The first `nextPageLoad` is the one that is actually supported by `flash` (after all the whole point is to remember data across a page load), the second `nextPageLoad` simulates an extra page load *after* the flash.

```
               +-----------+            +-----------+
               | PageLoad1 |            | PageLoad2 |
               +-----------+            +-----------+
                     |                        |
                     |                        |
-------------------\ |                        |
| Data is flashed. |-|                        |
|------------------| |                        |
                     |                        |
                     | Data can be read       |
                     |----------------------->|
                     |                        | -----------------------------\
                     |                        |-| Data is no longer readable |
                     |                        | |----------------------------|
                     |                        |
```